community/kernel/src/main/java/org/neo4j/kernel/api/exceptions/Status.java

```diff
@@ -121,6 +121,10 @@ public Code code()
         MarkedAsFailed( ClientError, "Transaction was marked as both successful and failed. Failure takes precedence" +
                 " and so this transaction was rolled back although it may have looked like it was going to be " +
                 "committed" ),
+        Outdated( TransientError, "Transaction has seen state which has been invalidated by applied updates while " +
+                "transaction was active. Transaction should succeed if retried." ),
+        LockClientStopped( TransientError, "Lock client is stopped, no more locks can be acquired." ),
```

Comment:
Too technical, I think. When and why are lock clients stopped? What is a user supposed to think when he gets this one popping out in a busy prod environment?